scroll-func.js

```diff
@@ -0,0 +1,7 @@
+let scrollBtn = document.querySelector(".back-to-top");
+window.addEventListener("scroll", function(){
+      let position = window.scrollY;
```

Comment:
@Asiedu13 Could you fix the indentations please, and make them uniform.

Reply:
Sure